src/project/ProjectModel.js

```diff
@@ -962,12 +962,6 @@ define(function (require, exports, module) {
 
         if (isFolder) {
             newPath += "/";
-        } else {
```

Comment:
does this also need a comment indicating that it's a CDO customization?

Reply:
no, the `if` condition was from upstream -- we just added the `else` condition